groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
```

Comment:
With all respect i have for this suggestion and wanted to implement myself but i feel i want get a minimal viable product first and for next phase of the project to have all the canvas call in a separate module in different github repo. 

Part of it is this is my first attempt to write the project python , vacation plans and project deadline i choose not have a separate library for integration